tests/positive/passwd/users.go

```diff
@@ -140,21 +141,96 @@ ENCRYPT_METHOD SHA512
 			},
 			Contents: "root:*::root\nusers:*::\nsudo:*::\nwheel:*::root,core\nsudo:*::\ndocker:*::core\nsystemd-coredump:!!::\nfleet:!!::core\nrkt-admin:!!::\nrkt:!!::core\ncore:*::\ntest:!::\njenkins:!::\n",
 		},
+		{
+			Node: types.Node{
+				Name:      "ignition",
+				Directory: "home/test/.ssh/authorized_keys.d",
+				User:      1000,
+				Group:     1000,
+			},
+			Contents: "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDBRZPFJNOvQRfokigTtl0IBi71LHZrFOk4EJ3Zowtk/bX5uIVai0Cd4+hqlocYL10idgtFBH28skeKfsmHwgS9XwOvP+g+kqAl7yCz8JEzIUzl1fxNZDToi0jA3B5MwXkpt+IWfnabwi2cRZhlzrz9rO+eExu5s3NfaRmmmCYrjCJIRPKSCrW8U0n9fVSbX4PDdMXVmH7r+t8MtR8523vCbakFR/Y0YIqkPVdfuUXHh9rDCdH4B7mt7nYX2LWQXGUvmI13mgQoy04ifkaR3ImuOMp3Y1J1gm6clO74IMCq/sK9+XJhbxMPPHUoUJ2EwbaG7Dbh3iqz47e9oVki4gIH stephenlowrie@localhost.localdomain\n",
```

Comment:
If the Ignition binary is built with a non-default `useAuthorizedKeysFile` setting, won't this fail?  We should probably pass the env var in this test too.

Reply:
:+1: - thanks for catching. Fixed this.